### ISSUE TITLE
Duplicate Increment stops at X.2

### ISSUE THREAD

**Issue by cartpauj** [barcradio/ultra-tracker#123]
Not sure if this is intentional or not. But I thought duplicates would continue incrementing X.2, X.3, X.4 etc. However, when I put in the same bib multiple times, it keeps X.2 everytime.

If this is by design, go ahead and close this issue out.

Screenshot: https://cdn.zappy.app/0165457d6f2972cad7e4428dceaf70f9.png

**Comment by thejosh69** [comment]
When checking for duplicates, it takes the inputted number and checks if it exists in the database. If the number exists, it adds 0.2 to the number before creating the record.  It doesn't validate that the x.2 number already exists.

**Comment by rleikis** [comment]
Yeah it doesn't keep incrementing it just does the. 2 as a single notation
so you could have a bunch of them.  We wanted something that would work
within a numeric field but be entirely apparent that something was wrong
with it.
Kind Regards,

Russ Leikis, KE7VFI
***@***.***

**Comment by rleikis** [comment]
> Could a 'duplicates' value be added to each runner in the database? When a new duplicate is determined, multipy the duplicates value by 0.1 and add to the inputted number. The update the value of the duplicate field by 1. This might also simplify the math for counting total number of duplicates?

Note that there is a Status property and database field for every time record so counting is very easy.

We want only one real/good time record of a bib number to ever exist.  This notation is to allow you to know which is the original and which are the imposters without requiring other special mechanisms.  

We considered incrementing this notation, but settled for just appending the same string rather than trying to keep track of how many of each duplicate there were.  The duplicates can all have the same bad bib. e.g. 14.2.

**Comment by rleikis** [comment]
We're pleased with where this is now with the Status tags making it easy to spot duplicates inline.  We'd like to keep the X.2 functionality as it stands today.

**Comment by Tibbs327** [comment]
After Discussing - It is functioning as intended, as Russ said, just want an indicator to show that there are duplicate information, regardless, each duplicate must be manually resolved and removed. OST has to match up with the bib numbers as they are assigned.
